Components/Snackbar/snackbar.js

```diff
@@ -0,0 +1,18 @@
+const baselineDemo = document.querySelector(".baselinedemo");
+const errorSnackbar = document.querySelector(".error");
+const leadingDemo = document.querySelector(".leadingdemo");
+const successSnackbar = document.querySelector(".success")
+
+baselineDemo.addEventListener("click", function baseLineHandler(){
+    errorSnackbar.style.cssText += 'display: flex; left: 34rem;bottom: 0.5rem;position: fixed;';
+    setTimeout(()=> {
+        errorSnackbar.style.display = "none";
+    },3000)
+});
+
+leadingDemo.addEventListener("click", function leadingHandler(){
+    successSnackbar.style.cssText += 'display: flex; left: 1rem;bottom: 0.5rem;position: fixed;';
```

Comment:
Instead of giving style in JS file add a class in the element and provide the style in the css for that class [ Single-responsibility principle ].

Reply:
Done! Thanks for the suggestion.